src/test/regress/input/external_table.source

```diff
@@ -284,14 +284,15 @@ CREATE PROTOCOL demoprot_non_superuser (readfunc = 'read_from_file', writefunc =
 CREATE TRUSTED PROTOCOL demoprot_non_superuser (readfunc = 'read_from_file', writefunc = 'write_to_file');
 RESET ROLE;
 
--- Test pg_exttable's encoding: QE's encoding should be consistent with QD
```

Comment:
Original commit does not contain any deletion!

Reply:
I brough it back along with my test in the latter commit. For some reason, cherry-picking also deletes those lines

Reply:
Brought back the deleted lines to that specific commit now